src/NuGet/Microsoft.CodeAnalysis.Compilers.nuspec

```diff
@@ -2,9 +2,13 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata minClientVersion="3.3">
     <id>Microsoft.CodeAnalysis.Compilers</id>
-    <summary>.NET Compiler Platform ("Roslyn")</summary>
+    <summary>
+      .NET Compiler Platform ("Roslyn").
+    </summary>
     <description>
-      .NET Compiler Platform ("Roslyn"). Install this package to get both C# and Visual Basic support. Install either of the dependencies directly to get one of the languages separately.
+      .NET Compiler Platform ("Roslyn").
+      Install this package to get both C# and Visual Basic support.
+      Install either of the dependencies directly to get one of the languages separately.
```

Comment:
This sentence is entirely correct and entirely impossible to read. "If you want just support for one language, install Microsoft.CodeAnalysis.CSharp or Microsoft.CodeAnalysis.VisualBasic".